### ISSUE TITLE
installing may break display settings

### ISSUE THREAD

**Issue by petebryan04** [jthomaschewski/i3-cinnamon#2]
i have an ultrawide monitor.

i have my nvidia drivers installed, but when i made the install, the resolution became terrible and insanely low (1024x768). and now i cant find a solution to reverse the problem.

xrandr cannot find my monitor either, it is seen as "unknown display".

**Comment by jthomaschewski** [comment]
Hi @petebryan04 ,

thanks for reporting this issue.

Does this problem only occur when you start the i3-cinnamon session?  
How have you installed this package? Arch AUR?

This package shouldn't make any changes to your system besides adding the session, so you should be able to just start a different session or uninstall this package as a last resort.

**Comment by pjbryan** [comment]
i was unable to boot into the computer at all. what happened was the screen went "bright" black, even though my computer was on. so i was not able to switch sessions to begin with.

edit: im on ubuntu

**Comment by jthomaschewski** [comment]
This doesn't really sound like it could be caused by this package.  
Only guess I have right now, is that maybe your Display Manager has issues with the xdesktop file https://github.com/jthomaschewski/i3-cinnamon/blob/master/session/i3-cinnamon-xsession.desktop

What Display Manager do you use? GDM?  
Which Ubuntu version are you on?  
Any errors in the log? E.g. `journalctl -xe`

And what are the steps you executed on your machine?

Guess:
1. Git clone & make install
2. Reboot?
3. Black screen - no login window shown?

Is this correct?  
It would be helpful if you could provide more information. I would then try to reproduce your issue.

**Comment by petebryan04** [comment]
Yes your guess is correct. I just git cloned and made the install. I then proceeded to reboot, no login window was shown. Just a bright black screen.
Im on ubuntu 22.10. I use the default display manager for ubuntu cinnamon.
I also have a 3440x1440 single monitor connected.

**Comment by pjbryan** [comment]
Yes your guess is correct. I just git cloned and made the install. I then proceeded to reboot, no login window was shown. Just a bright black screen.
Im on ubuntu 22.10. I use the default display manager for ubuntu cinnamon.
I also have a 3440x1440 single monitor connected.
